kvcounter/rust/wadm.yaml

```diff
@@ -0,0 +1,46 @@
+# This is a full example of how to run the kvcounter actor exposed with an HTTP server. Using this
+# example requires you to have a Redis server running locally (though the linkdef can be modified to
+# use a Redis server you have running elsewhere) and WADM running:
+#
+# https://github.com/wasmCloud/wadm/tree/main/wadm. You can deploy this example with a simple command:
+#
+# `wash app deploy wadm.yaml`
+
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  name: {{project-name}}
+  annotations:
+    version: v0.0.1
+    description: "wasmCloud Key Value Counter Example"
+spec:
+  components:
+    - name: kvcounter
+      type: actor
+      properties:
+        image: ghcr.io/{{github.REPOSITORY}}:{{project-name}}:0.1.0
+      traits:
+        - type: spreadscaler
+          properties:
+            replicas: 1
+        - type: linkdef
+          properties:
+            target: redis
+            values:
+              URL: redis://127.0.0.1:6379/
+        - type: linkdef
+          properties:
+            target: httpserver
+            values:
+              ADDRESS: 0.0.0.0:8081
+
+    - name: httpserver
+      type: capability
+      properties:
+        image: wasmcloud.azurecr.io/httpserver:0.17.0
+        contract: wasmcloud:httpserver
+    - name: redis
+      type: capability
+      properties:
+        image: wasmcloud.azurecr.io/kvredis:0.21.0
```

Comment:
Do you want me to change this to builtin?

Reply:
And if so, can we do it later?

Reply:
We can do it later if we need to. It wouldn't be that hard to swap this out I think though